google-cloud-language/README.md

```diff
@@ -7,17 +7,14 @@ recognition, and text annotations.
 - [Client Library Documentation][]
 - [Product Documentation][]
 
-*The 0.28.0 release introduced breaking changes relative to the previous
```

Comment:
Do we want to keep the 0.28.0 migration notice?

Reply:
What do you think is a reasonable amount of time to keep it for? It's been available for two months, and it's going to get nuked on every GAPIC refresh.

Reply:
> What do you think is a reasonable amount of time to keep it for?

I have no idea. I just wasn't sure if removing it was a conscious decision or an oversight.

@quartzmo Any input on how long to keep the notice?

Reply:
I also have no idea. I'd keep it for a few more versions if it's not too much trouble. Maybe @frankyn has an opinion, since he wrote the migration guide?

Reply:
(Thanks @quartzmo for the ping). I'm not sure how long we should have the migration guide in place. I would keep the migration guide link in documentation so users have a direct way of finding it when reading the documentation. Is there a better way to keep this in docs after each refresh?

Given that this is being removed after each refresh. Is the post-install message also being removed?

Reply:
> Is there a better way to keep this in docs after each refresh?

No, it would have to be done manually. In our discussions about the migration, I had originally suggested that this should go away on the first refresh, but since the DPE team is assuming ownership of the libraries' manual components, I'm happy to follow your guidance here.

> Given that this is being removed after each refresh. Is the post-install message also being removed?

Correct.

Reply:
Thanks @geigerj. My preference is to keep it in the docs and the post-install message

Summoning @jonparrott for visibility and input on this case.

Reply:
Do y'all keep a `CHANGELOG`/`NEWS` or some other release log? If so, it's probably fine to just keep this notice in the `0.28.0` section of whatever that log is. It certainly doesn't need to be in the readme as we shed legacy towards v1.0.0.

Reply:
@jonparrott Yes, the migration guide is linked to from the CHANGELOG.